# -*- coding: utf-8 -*-
#
# Copyright (c) 2020, CloudBlue
# All rights reserved.
#

from connect.client import ClientError


def generate(client, parameters, progress_callback):
    raise RuntimeError('Custom error')
